force-app/main/service/DonationHistoryService.cls

```diff
@@ -35,16 +35,12 @@
 public with sharing class DonationHistoryService {
     
     static final String NPSP_CREDIT_ROLES = 'NPSP_OppsSoftCreditRoles';
```

Comment:
This constant is unused and we should remove it.

Reply:
constant removed!